apps/sms/js/activity_picker.js

```diff
@@ -76,13 +76,6 @@ var ActivityPicker = {
       }
     }), onsuccess, onerror);
   },
-  sendMessage: function ap_sendMessage(number) {
```

Comment:
Basically after I adressed your suggestion it's clear that ```sendMessage``` should not be in activity picker anymore :)